src/Modules/ConfigOptions.lua

```diff
@@ -744,6 +744,9 @@ Huge sets the radius to 11.
 	{ var = "buffAdrenaline", type = "check", label = "Do you have Adrenaline?", tooltip = "This will enable the Adrenaline buff, which grants:\n\t100% increased Damage\n\t25% increased Attack, Cast and Movement Speed\n\t10% additional Physical Damage Reduction", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:Adrenaline", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
+	{ var = "changedStanceLastSecond", type = "check", label = "Changed Stance in the last 1s?", tooltip = "'Changing Stance' occurs by activating a Stance skill while it's toggled on", apply = function(val, modList, enemyModList)
```

Comment:
this should have ifCond for StanceChangeLastSecond

Reply:
ifCond is for conditionally rendering the field? So i'm assuming I would need to have something setup in the ModParser that flips some conditional to render this config option only if it's truthy?

Reply:
![image](https://user-images.githubusercontent.com/5115805/229313660-d0b482e2-19a2-4bbc-af2f-c67c70cfa61e.png)

it should react to this I think, its for type = "Condition" checks